src/sentry/tasks/commit_context.py

```diff
@@ -26,6 +33,24 @@
 logger = logging.getLogger(__name__)
 
 
+def queue_comment_task_if_needed(commit: Commit, group_owner: GroupOwner):
+    from sentry.tasks.integrations.github.pr_comment import comment_workflow
+
+    pr = PullRequest.objects.filter(
+        organization_id=commit.organization_id, merge_commit_sha=commit.key
+    )
+    if (
+        pr.exists()
+        and pr[0].date_added >= datetime.now(tz=timezone.utc) - timedelta(days=30)
```

Comment:
Would pr ever include more than one object? and if so why do we choose pr[0] over the others?

Reply:
`pr` is technically a query so we might want to call it `pr_query`. we could also use `get()` but `get()` throws an error if the item doesn't exist or if multiple items exist that match the criteria